### PR TITLE
First PR to test the workflow

### PR DESCRIPTION
**Pull request by VasilValtchev**
### Introduction
This PR is to be used to test the workflow and comments
This line is part of the PR description and is added later.

### CODE REVIEW

**Comment by cumulus5** [comment]
This is a comment related to the whole PR

**Comment by VasilValtchev** [comment]
This is also a comment related to the whole PR (standalone not a reply on a previous comment)

**Comment by VasilValtchev** [comment]
> 
> 
> This is a comment related to the whole PR

This is a reply to the first comment from cumulus5

File: dir4/file1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is file1 in dir4
```
Comment (cumulus5) [inline dir4/file1.txt:1]: This is a comment on the code line.
Maybe you should add more lines to show how a snipped looks like

Comment (VasilValtchev) [reply dir4/file1.txt:1]: Done.

File: dir4/file1.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+First added line
+Second added line
+Third added line
+Forth added line
```
Comment (cumulus5) [inline dir4/file1.txt:6]: This comment is for three lines, line 2-4

**Review by cumulus5 — CHANGES_REQUESTED** [review]
Add one more file to dir4 with at least 6 lines

**Comment by VasilValtchev** [comment]
> 
> 
> Add one more file to dir4 with at least 6 lines

Done

**Review by cumulus5 — APPROVED** [review]
Everything looks good

**Pull request merged**